pgml-dashboard/static/js/docs-toc.js

```diff
@@ -4,15 +4,23 @@ import {
 
 export default class extends Controller {
   connect() {
-    this.scrollSpyAppend();
-  }
-
-  scrollSpyAppend() {
     const spy = new bootstrap.ScrollSpy(document.body, {
```

Comment:
I'm surprised this doesn't get garbage collected immediately. Perhaps the plugin attaches itself to the element and exists until that element is removed.